src/styles/_catalog.scss

```diff
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

Comment:
you are using that value twice, move it to var

Reply:
also specify for which prop you're applying that transition duration

Reply:
> you are using that value twice, move it to var

Didn`t manage with $transitionDuration: 0.3s; (mistakes in any case, have made more than 10 attempts)
![hover4](https://user-images.githubusercontent.com/93187497/184856959-38771a97-a8b2-4360-8624-9fd748ecf4f2.JPG)
![hover3](https://user-images.githubusercontent.com/93187497/184856971-0a729104-dbed-444e-8ced-21c23217a8be.JPG)
![hover2](https://user-images.githubusercontent.com/93187497/184856973-dbe09e56-e57a-46f1-9f4d-0f24bd66ce03.JPG)
![hover1](https://user-images.githubusercontent.com/93187497/184856975-82ceb2c3-accc-4c01-b370-c004aef386b5.JPG)

Reply:
> also specify for which props you're applying that transition duration

done

Reply:
imports vars before blocks

Reply:
> imports vars before blocks

It works!!!! 👍